### PR TITLE
Add generate list of exercises of specific difficulties

### PR DESCRIPTION
**Pull request by ChubbsBunns**
Updated features to include generating exercises of a certain level of difficulty
Added dylan.md file
Created test cases that test for correct level of difficulty of tasks
Created test cases that incorporates the pre-existing gym and static with difficulty tasks
Created a help function showing the user all available commands

### CODE REVIEW

File: src/main/java/seedu/duke/exercisegenerator/GenerateExercise.java [hunk 1]
```diff
@@ -1,66 +1,113 @@
...
 public class GenerateExercise {
 
-    private ArrayList<ExerciseData> exerciseData = new ArrayList<>();
+    private static final String EASY = "easy";
+    private static final String MEDIUM = "medium";
+    private static final String HARD = "hard";
+
+    private static final String BEGINNER = "beginner";
+    private static final String INTERMEDIATE = "intermediate";
+    private static final String EXPERT = "expert";
```
Comment (EangJS) [inline src/main/java/seedu/duke/exercisegenerator/GenerateExercise.java:17]: Actually do we need two different sets ?

File: src/main/java/seedu/duke/util/OuputText.java [hunk 2]
```diff
@@ -0,0 +1,2 @@
+package seedu.duke.util;public class OuputText {
+}
```
Comment (EangJS) [inline src/main/java/seedu/duke/util/OuputText.java:2]: What does this class do ah?

File: src/main/java/seedu/duke/exercisegenerator/GenerateExercise.java [hunk 3]
```diff
@@ -1,66 +1,113 @@
...
+import seedu.duke.errors.DukeError;
 import seedu.duke.exersisedata.ExerciseData;
 
 import java.util.ArrayList;
 import java.util.Random;
 
 public class GenerateExercise {
 
-    private ArrayList<ExerciseData> exerciseData = new ArrayList<>();
+    private static final String EASY = "easy";
```
Comment (L-K-Chng) [inline src/main/java/seedu/duke/exercisegenerator/GenerateExercise.java:11]: Great job changing to constants instead of keeping as magic strings.

**Review by L-K-Chng — APPROVED** [review]
Good addition of "easy", "medium" and "hard" list of exercises.

**Pull request merged**